index.js

```diff
@@ -87,7 +87,14 @@ function Log(options) {
 
   if (options.rawTransports){
     logTransports = options.transports
-  } else {
+  }
+  else if(options.transports.length == 0){
+    //defaults to console if no transports are given
+    logTransports = new winston.transports.Console({
+        format: winston.format.simple()
```

Comment:
Nit: I would lean towards `winston.format.json()` since we use that throughout almost all our repos. But I don't feel strongly about it, since I wouldn't expect us to be using this default transport much anyway.